recipes/sleapyfaces/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "sleapyfaces" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sleapyfaces-{{ version }}.tar.gz
+  sha256: 071f08edcd5a8ed7a51f9b8482e95c8d1573fabb0976d09bced3c4dcc635a0f7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - hatchling >=1.11.0
+    - hatch >=1.6.0
+    - matplotlib-base >=3.5.0
+    - ffmpeg-python >=0.2.0
+    - hdf5 >=1.12.0
+    - h5py >=3.7.0
+    - numpy >=1.24.0
+    - numexpr >=2.8.0
+    - pytables >=3.6.0
+  host:
+    - python >=3.7
+    - pip >=22.0.0
+    - hatchling >=1.11.0
+    - hatch >=1.6.0
+  run:
+    - python >=3.7
+    - click >=8.0.3
+    - hatchling >=1.11.0
+    - hatch >=1.6.0
+    - ffmpeg-python >=0.2.0
+    - hdf5 >=1.12.0
+    - h5py >=3.7.0
+    - numpy >=1.24.0
+    - numexpr >=2.8.0
+    - pandas >=1.5.0
+    - python-rapidjson >=1.9
+    - scikit-learn >=1.2.0
+    - scipy >=1.9.0
+    - pytables >=3.6.0
+    - matplotlib-base >=3.5.0
+
+test:
+  imports:
+    - sleapyfaces
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/sleapyfaces/
+  summary: A package for extracting facial expressions from SLEAP analyses
+  description: |
+    sleapyfaces is a data analysis package for extracting
+    facial expressions of mice from SLEAP analyses. It is
+    designed to work with the SLEAP software package, which
+    provides a graphical user interface for annotating
+    animal behavioral videos. SLEAP is available at
+    https://sleap.ai. This package also depends on many
+    assumptions about the data format and structure of the
+    SLEAP analyses. It is not intended to be a general
+    tool for extracting facial expressions from SLEAP
+    analyses, but rather a tool for extracting facial
+    expressions from the specific data format and structure
+    used in the lab of Dr. Kay Tye at The Salk Institute
+    for Biological Studies.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  doc_url: https://github.com/annie444/sleapyfaces
+  dev_url: https://github.com/annie444/sleapyfaces
```

Comment:
You link your own fork here but are using the official version, which one is correct?